packages/react-core/src/components/NotificationDrawer/NotificationDrawerGroup.tsx

```diff
@@ -34,7 +34,10 @@ export const NotificationDrawerGroup: React.FunctionComponent<NotificationDrawer
   title,
   ...props
 }: NotificationDrawerGroupProps) => (
-  <section {...props} className={css(styles.notificationDrawerGroup, isExpanded && styles.modifiers.expanded)}>
```

Comment:
What does this have to do with the tabs?  Is it just fixing a linting issue?

Reply:
Yep! `className` should clearly be added there, but was removed at some point.